internal/handlers/common/projection.go

```diff
@@ -35,7 +35,7 @@ var errProjectionEmpty = errors.New("projection is empty")
 func ValidateProjection(projection *types.Document) (*types.Document, bool, error) {
 	validated := types.MakeDocument(0)
 
-	if projection == nil {
+	if projection.Len() == 0 {
```

Comment:
What if projection is not set at all? Shouldn't it be `if projection == nil || projection.Len() == 0`?
Or `projection == nil` is not possible?

Reply:
https://github.com/FerretDB/FerretDB/blob/a1aee0233ffbf7ea170f760d9189618a02e145c8/internal/types/document.go#L131-L135